envi/archs/ppc/regs.py

```diff
@@ -313,21 +313,21 @@ def __init__(self):
 
 # The general registers also need a few system and special registers
 regs_general.extend([reg for reg, size in sysregs])
-regs_general.append('lr')
-regs_general.append('ctr')
-regs_general.append('xer')
+regs_general.append('LR')
+regs_general.append('CTR')
+regs_general.append('XER')
 
 regs_fpu = ['f%d' %x for x in range(32)]
-regs_fpu.append('fpscr')
+regs_fpu.append('FPSCR')
 
 regs_altivec = ['vr%d' %x for x in range(64)]
-regs_altivec.append('vscr')
-regs_altivec.append('vrsave')
+regs_altivec.append('VSCR')
+regs_altivec.append('VRSAVE')
 
 regs_spe = ['ev%dh' %x for x in range(32)]
 regs_spe.extend([reg for reg, size in sysregs])
-regs_spe.append('acc')
-regs_spe.append('spefscr')
+regs_spe.append('ACC')
+regs_spe.append('SPEFSCR')
```

Comment:
i'm pretty sure we went through some work to get these all lower case.  any functional reason to change them back?